articles/introduction-to-kubernetes/index.md

```diff
@@ -0,0 +1,121 @@
+Kubernetes is a framework that runs containerized apps across a cluster of machines and coordinates them. It is designed to manage the lifecycle of containers and services providing scalability, predictability, and availability.
```

Comment:
@BrianaNzivu  

You can leave you comments and suggestions via these in-line comments.